get-covid-data/main.sh

```diff
@@ -1,23 +1,25 @@
-
 #!/usr/bin/env sh
 
-# Compute pi.
-#
 # Input:
 #     --output  Output bucket/folder
 #     --params  Json containing integer "seed"
 #
-# Output:
-#     in.json and out.json
-#
-#     out.json has as "result" field, which is 0 or 1, depending
-#     on whether the random point is inside or outside the circle.
-#
-# Blair Drummond
-#
+
+# Get the source
+git clone --depth=1 \
+    https://github.com/kennethchu-statcan/covid19/ \
+    /tmp/covid19 || exit 1
```

Comment:
Might be worth either tossing an error message on failure or choosing an error code that you will recognize down the road as this type of failure. I could see someone pulling out there hair wondering what's wrong with their code when it was just a network error. 😄

Reply:
Yes, especially with Istio that's a good idea :-) Point taken haha